dev/buildMinimal.dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM montagu-reverse-proxy-shared-build-env
```

Comment:
I don't understand why a docker-in-docker approach seems to being used here

Reply:
Docker in docker is because we have npm installed in the build container.